jupyterhub_singleuser_profiles/ui/src/CustomElements/VarForm.js

```diff
@@ -39,22 +41,28 @@ class VarForm extends React.Component {
     }
 
     enterVariable(event) {
-        this.setState({key: event.target.text}, () => {this.checkKey(this.state.key)})
+        this.setState({key: event.target.text})
```

Comment:
You the argument for setState is dictionary, you don't need to call it twice, just use both keys in the dictionary